deploy/helm/kubecf/values.yaml

```diff
@@ -515,3 +516,13 @@ bits:
   tls_cert_name: certificate
   tls_key_name: private_key
   tls_ca_name: ca
+
+eirinix:  
+  persi-broker:
+    service-plans:
+    - id: default
+      name: "default"
+      description: "Existing default storage class"
+      kube_storage_class: "standard"
```

Comment:
This does not belong here: `standard` is for minikube, but this will get deployed with the chart we ship to customers.

(The proper fix, I think, is to drop it in the upstream (also our) chart, and use the default storage class.)

Reply:
@mook-as I will change it back to "default"

Reply:
@mook-as I also agree with the "proper" fix - it **should** use the `default` storage class.
Can you please add an issue for that?